pkg/registry/registrytest/namespace.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registrytest
```

Comment:
I haven't looked into what this file is used for yet, but I'll question its necessity anyway.

I'll give it a look, but could you also investigate whether or not we actually need to do add all of this? I'd be surprised if adding a new field required all of this, but I could be wrong.